neo4j/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# integration pip requirements
+hashlib
+httplib2==0.9.2
```

Comment:
We currently have a version mismatch here, this is already defined in `dd-agent`, see error logs in the CI for more details:
```
There's a version mismatch with httplib2 ==0.9
```

Do you need this to be 0.9.2? Also, we should pin hashlib to a specific version (not sure if we're using it elsewhere), and on top of it I think hashlib is having problems installing/building in Travis 😞 

I'll try to take a look see if I can spot what the issue is.